### PR TITLE
added the tutorial for the re-factored configuration

### PR DESCRIPTION
**Pull request by yxdyc**
as the title said.

### CODE REVIEW

File: _docs/02-start-your-own-case.md [hunk 1]
```diff
@@ -229,3 +230,61 @@ register_metric("mymetric", call_my_metric)
 
 
 -  Step3: put this `.py` file in the `federatedscope/contrib/metircs/` folder, and add `"mymetric"` to `cfg.eval.metric` activate it. 
+
+
+## <span id="config">Specify your own configuration</span>
+
+### Basic usage
+FederatedScope provides an extended configuration system based on [yacs](https://github.com/rbgirshick/yacs). We leverage a two-level tree structure that consist of several internal dict-like containers to allow simple key-value access and management. For example,
```
Comment (xieyxclack) [inline _docs/02-start-your-own-case.md:238]: consist -> consists

File: _docs/02-start-your-own-case.md [hunk 2]
```diff
@@ -229,3 +230,61 @@ register_metric("mymetric", call_my_metric)
...
+cfg.backend = 'torch'  # level-1 configuration
+
+cfg.federate = CN()  # level-2 configuration
+cfg.federate.client_num = 0
+```
+The frequently-used APIs include
+- `merge_from_file`, `merge_from_other_cfg` and `merge_from_list` that load configs from a yaml file, another `cfg` instance or a list stores the keys and values.
+- Besides, we can use `freeze` to make the configs immutable and save the configs in a yaml file under the specified `cfg.outdir`.
+- Both these functions will trigger the configuration validness checking.
+- To modify a config node after calling `freeze`, we can call `defrost`.
```
Comment (xieyxclack) [inline _docs/02-start-your-own-case.md:249]: Shall we mention the `init_cfg` here? And show an example of these APIs?

**Review by xieyxclack — COMMENTED** [review]
Looks good to me, and please refer to the inline comments for minor suggestions. Thanks!

**Comment by yxdyc** [comment]
I have pushed a new request to reflect the suggestions from yuexiang.

File: _docs/02-start-your-own-case.md (hunk 2 above)
Comment (yxdyc) [reply _docs/02-start-your-own-case.md:249]: See line 251~279

**Review by FederatedScopeTeam — APPROVED** [review]
approve

**Pull request merged**